clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/remote/RemoteCacheContainerBuilder.java

```diff
@@ -78,18 +78,19 @@ public RemoteCacheContainerBuilder configure(OperationContext context, ModelNode
             Configuration configuration = this.configuration.getValue();
             RemoteCacheManager remoteCacheManager = new RemoteCacheManager(configuration);
             remoteCacheManager.start();
-            InfinispanLogger.ROOT_LOGGER.debugf("%s remote cache container started", this.name);
+            InfinispanLogger.ROOT_LOGGER.remoteCacheContainerStarted(this.name);
             return remoteCacheManager;
         };
 
         Consumer<RemoteCacheManager> destroyer = remoteCacheManager -> {
             remoteCacheManager.stop();
-            InfinispanLogger.ROOT_LOGGER.debugf("%s remote cache container stopped", this.name);
+            InfinispanLogger.ROOT_LOGGER.remoteCacheContainerStopped(this.name);
         };
 
         Service<RemoteCacheContainer> service = new SuppliedValueService<>(mapper, supplier, destroyer);
         ServiceBuilder<RemoteCacheContainer> builder = target.addService(this.getServiceName(), service)
-                .setInitialMode(ServiceController.Mode.ON_DEMAND)
+                // Workaround org.infinispan.client.hotrod.RemoteCacheContainer not being restartable
```

Comment:
We need to figure out why this is and come up with a workaround.

Reply:
Fixed now with https://issues.jboss.org/browse/ISPN-8918 (thanks!)